jupyter-extension/jupyterlab_pachyderm/env.py

```diff
@@ -9,7 +9,8 @@
 SIDECAR_MODE = strtobool(os.environ.get("SIDECAR_MODE", "False").lower())
 MOUNT_SERVER_LOG_DIR = os.environ.get("MOUNT_SERVER_LOG_DIR") # defaults to stdout/stderr
 NONPRIV_CONTAINER = os.environ.get("NONPRIV_CONTAINER") # Unset assume --privileged container
```

Comment:
Looks like this should perhaps be more like `PACHYDERM_EXT_DEBUG` so it is really a bool and not just a string.

`strtobool(os.environ.get("NONPRIV_CONTAINER", "False").lower())`